src/backend/src/controllers/reimbursement-requests.controllers.ts

```diff
@@ -29,6 +29,30 @@ export default class ReimbursementRequestsController {
     }
   }
 
+  static async editReimbursementRequest(req: Request, res: Response, next: NextFunction) {
+    try {
+      const { id } = req.params;
+      const { dateOfExpense, vendorId, account, expenseTypeId, totalCost, reimbursementProducts, saboId, receiptPictures } =
+        req.body;
+      const user = await getCurrentUser(res);
+      const updatedReimbursementRequestId = await ReimbursementRequestService.editReimbursementRequest(
+        id,
+        dateOfExpense,
+        vendorId,
+        account,
+        expenseTypeId,
+        totalCost,
+        reimbursementProducts,
+        saboId,
```

Comment:
the sabo id isn't in the edit reimbursement request form, pls remove it

Reply:
do we not want it in the form?